package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "webtorrent",
   "description": "Streaming torrent client",
-  "version": "0.108.6",
+  "version": "0.108.7",
```

Comment:
If you don't mind, please undo this change as it will be automatically cincremented with the release process we use 😉